test/regression/test/netxtestcontrol.c

```diff
@@ -204,6 +204,9 @@ void    netx_dest_table_add_fail_test_application_define(void *);
 
 void    netx_ip_basic_test_application_define(void *);
 void    netx_ip_link_status_test_application_define(void *);
+void    netx_ip_link_status_test2_application_define(void *);
+void    netx_shaper_cbs_test_application_define(void *);
```

Comment:
1. netx_shaper_tas_test is not added, 2. could we not inset the tsn test case in netxduo test cases? we may relocate the it,  such as:

```
    /* TSN test */
    {netx_shaper_cbs_test_application_define, TEST_TIMEOUT_LOW},
    {netx_mrp_state_machine_test_application_define, TEST_TIMEOUT_LOW},
```